dags/veda_data_pipeline/veda_vector_pipeline.py

```diff
@@ -74,9 +74,9 @@ def ingest_vector_task(payload):
 
 
 @task
-def invalidate_cloudfront(ti):
+def invalidate_cloudfront(event: dict={}):
     import boto3
-    if not ti.dag_run.conf['invalidate_cloudfront']:
+    if not event.get("invalidate_cloudfront", False):
```

Comment:
```suggestion
    if not event.get("invalidate_cloudfront"):
```

Should agree with `template_dag_run_conf`--defaulting False here is confusing although it is the default I'd prefer 😹